spyder/config/main.py

```diff
@@ -94,7 +94,6 @@
               'use_custom_margin': True,
               'custom_margin': 0,
               'use_custom_cursor_blinking': False,
-              'custom_cursor_blinking': 1000,

```

Comment:
@ccordoba12 and @jnsebgosselin I think we still need to have the option here, the only difference is that it would be `None` by default, and when building the spinbox, we would check if it is None and update with the default by QApplication, or just use the stored value.